disruption_generator/trigger/trigger.py

```diff
@@ -28,6 +29,31 @@ def __getattr__(self, item):
                 return False
 
     async def restart_service(self):
+        """
+        Restart a service on a remote host
+
+        Returns:
+            result (bool): Result of the restart
+        """
         cmd = "systemctl restart"
         result = await self.run_client(cmd)
         return result
+
+    async def latency(self):
+        """
+        Add latency to a remote hosts link
+
+        Returns:
+            result (bool): True if successful, False otherwise
+        """
+        cmd_add = "tc qdisc add dev eth0 root netem delay"
+        cmd_del = "tc qdisc del dev eth0 root netem delay"
+        result = await self.run_client(cmd_add)
+        wait = self.action.wait if self.action.wait else 10
```

Comment:
I think this is an use case for [getattr](https://docs.python.org/3/library/functions.html#getattr). Also, I thank that the default wait time (10) should be either configurable or better expose. At least put it to coroutine docstring pls.